fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryManager.java

```diff
@@ -73,6 +87,70 @@ public synchronized void close() throws IOException {
         createdDirectories.clear();
     }
 
+    @SuppressWarnings("PMD.CloseResource")
+    public CompletableFuture<Void> mergeIndex(LuceneAnalyzerWrapper analyzerWrapper) {
+
+        final ScanProperties scanProperties = ScanProperties.FORWARD_SCAN.with(
```

Comment:
It's possible that the root expression does not have any grouping keys, in which case you should just provide an empty Tuple for the grouping key.

Reply:
✅